natvis/cppwinrtvisualizer.vcxproj

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="packages\Microsoft.Windows.WinMD.1.0.191006.1\build\native\Microsoft.Windows.WinMD.props" Condition="Exists('packages\Microsoft.Windows.WinMD.1.0.191006.1\build\native\Microsoft.Windows.WinMD.props')" />
+  <Import Project="packages\Microsoft.Windows.WinMD.1.0.210225.2\build\native\Microsoft.Windows.WinMD.props" Condition="Exists('packages\Microsoft.Windows.WinMD.1.0.210225.2\build\native\Microsoft.Windows.WinMD.props')" />
```

Comment:
Should we update cppwinrt as well?

Reply:
Updated it. Shouldn't have any effect, but it would be good hygiene to keep these in sync.